.gitignore

```diff
@@ -75,3 +75,4 @@ terraform/aks/vendor/
 Procfile.dev
 missing-location-graph-from-dttp.json
 .vscode/*
+Caddyfile
```

Comment:
We used this in Publish as a reverse proxy for local dev with SSL. Just adding this here for consistency as I use it to do the same for Register